corehq/apps/cloudcare/templates/formplayer/multi_select_continue_button.html

```diff
@@ -1,6 +1,10 @@
 {% load i18n %}
 <button type="button" class="btn btn-success btn-lg formplayer-request multi-select-continue-btn" disabled="true">
-  {% trans "Continue" %}
+  <% if (selectText && selectText.length > 0) { %>
+    <%= selectText %>
```

Comment:
`<%-` to get HTML escaping, since this should only be text

Reply:
ah, I hadn't considered this. I'm looking at the docs and it seems to be the other way around?
```<%= Outputs the value into the template (HTML escaped)```
```<%- Outputs the unescaped value into the template```

https://ejs.co/#docs:~:text=%3C%25%3D%20Outputs%20the%20value,value%20into%20the%20template

Reply:
By default backbone uses underscore templates, which (annoyingly) are the opposite: https://underscorejs.org/#template

I think you can use EJS with backbone, but we don't.

Reply:
cool, thanks for the clarification.